examples/make.mk

```diff
@@ -80,5 +80,9 @@ CFLAGS += \
 ifeq ($(DEBUG), 1)
   CFLAGS += -O0 -ggdb -DCFG_TUSB_DEBUG=1
 else
+ifneq ($(BOARD), spresense)
```

Comment:
does **spresense** have issue with `lto` option. Do you know what could cause the option failed on this port

Reply:
Spresense uses the static library (libsdk.a) that was built without this flag. As far as I know, lto requires that all source files use this flag. For this reason, Spresense cannot use this flag.

Reply:
Ah I see, LTO require all sources must be compile with the flags. Linking with precompiled library without lto can be troublesomes. I may modify makefile to make this as option for `board.mk`, it will be a cleaner solution.